dockerfiles/testing-environment/Laravel94/docker-scripts/local-run.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "Starting up local testing supervisord."
+echo "Tracer to be installed: ${TRACER_DOWNLOAD_URL}"
+
+if [ ! -z ${TRACER_DOWNLOAD_URL} ]; then
+    echo "Installing tracer at: ${TRACER_DOWNLOAD_URL}"
+    wget -O dd-trace-php.deb ${TRACER_DOWNLOAD_URL}
+    dpkg -i dd-trace-php.deb
+fi
```

Comment:
How does the profiler get installed? I think we need to use `datadog-setup.php` and check for a profiling env var?

Also, does that URL get overwritten by the relenv jobs?

Reply:
That's the neat part 🤯 
Basically in the [bazel deployment](https://github.com/DataDog/datadog-reliability-env/blob/master/apps/php/laravel/deployment.yaml) we add another layer with the [`run.sh`](https://github.com/DataDog/datadog-reliability-env/blob/master/apps/php/laravel/run.sh) and overwrite the `cmd` of the container. It's not nice, but this is just copied over from the other profiling containers in relenv.